tests/codeception/functional/Install/Content/VideoEmbedCest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use Faker\Factory;
+
+/**
+ * Class VideoEmbedCest.
+ *
+ * @group install
+ */
+class VideoEmbedCest {
+
+  /**
+   * Faker service.
+   *
+   * @var \Faker\Generator
+   */
+  protected $faker;
+
+  /**
+   * Test constructor.
+   */
+  public function __construct() {
+    $this->faker = Factory::create();
+  }
+
+  /**
+   * I can create a page with a video embed and verify caption.
+   */
+  public function testVideoEmbed(FunctionalTester $I) {
+    // Login and add flexible page
+    $I->logInWithRole('contributor');
+    $I->amOnPage('node/add/hs_basic_page');
+    $I->fillField('Title', $this->faker->words(3, TRUE));
+
+    // Add text field
+    $I->scrollTo('#edit-field-hs-page-components-add-more');
+    $I->click('List additional actions', '#edit-field-hs-page-components-add-more');
+    $I->scrollTo('#field-hs-page-components-hs-text-area-add-more');
+    $I->click('#field-hs-page-components-hs-text-area-add-more');
+    $I->wait(2);
+
+    // Add media
+    $I->click('.cke_button__drupalmedialibrary');
+    $I->waitForText('Add or select media');
+    $I->click('.media-library-menu__link[data-title="Video"]');
+    $I->waitForText('Add Video via URL');
+    $I->fillField('Add Video via URL', 'https://www.youtube.com/watch?v=95N_spFNEkY');
+    $I->click('.js-form-submit[value="Add"]');
```

Comment:
You can use `$I->click('Add', '.js-form-submit')` here instead.

Reply:
Changed.